modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2502,11 +2503,13 @@ public void testGetObjectEntry() throws Exception {
 
 	@Test
 	public void testGetValuesList() throws Exception {
+		Sort[] sortById = {new Sort("id", false)};
```

Comment:
Hi @icgsbr, 
I would just rename this to `sorts`, because that is all I could find in the examples in the portal. Did you find any examples with different names?